docusaurus.config.js

```diff
@@ -524,6 +524,15 @@ const config = {
         insights: false,
         contextualSearch: true,
       },
+      mermaid: {
+        theme: { light: "base", dark: "forest" },
+        options: {
+          themeVariables: {
+            primaryColor: "#b2dfdb",
+            primaryBorderColor: "#00695c",
+          },
+        },
+      },
```

Comment:
Nice. I assume these come from the site theme. A comment might be welcome here.

Also, I noticed in dark mode the diagrams are not really visible, so we might need some adjustments here:

![image](https://github.com/conda-forge/conda-forge.github.io/assets/2559438/9a07c75f-c302-4ad9-8aad-4652100949b1)

Reply:
And while you are at it, we can also change the font family to match the site's. [Apparently these are the options](https://github.com/mermaid-js/mermaid/issues/1614#issuecomment-859462043).